src/ingestors/foundation/index.ts

```diff
@@ -0,0 +1,133 @@
+import { MintContractOptions, MintIngestor, MintIngestorResources } from '../../lib/types/mint-ingestor';
+import { MintIngestionErrorName, MintIngestorError } from '../../lib/types/mint-ingestor-error';
+import { MintInstructionType, MintTemplate } from '../../lib/types/mint-template';
+import { MintTemplateBuilder } from '../../lib/builder/mint-template-builder';
+import { getFoundationMintPriceInWei } from './onchain-metadata';
+import { getFoundationMintByAddress } from './offchain-metadata';
+import { FOUNDATION_MINT_ABI } from './abi';
+
+const CONTRACT_ADDRESS = '0x62037b26ffF91929655AA3A060F327b47d1e2b3e';
+
+const getChainId = (chain: string) => {
+  switch (chain) {
+    case 'base':
+      return 8453;
+    case 'eth':
+    default:
+      return 1;
+  }
+};
+
+export class FoundationIngestor implements MintIngestor {
+  async supportsUrl(_resources: MintIngestorResources, url: string): Promise<boolean> {
```

Comment:
This returns true and then errors for this URL:
https://foundation.app/mint/eth/0xB043f48B7C2FA1bfaA03D99C00b36A2A3B05b9e2

Can we filter the supportsUrl to the greatest extent possible to those that we believe it will succeed with?